plugins/www/Optimove.js

```diff
@@ -11,6 +11,28 @@ document.addEventListener("deviceready", init, false);
 document.addEventListener("resume", resume, false);
 document.addEventListener("pause", pause, false);
 
+function isString(val) {
```

Comment:
if we convert this to typescript many of type checks will become redundant. But still can keep checking for "", for example